### DISCUSSION TITLE
Logo proposal

### DISCUSSION THREAD

**Discussion by jorlogicus** [kickstarter/rack-attack#532] [Ideas & Proposals]
Hello there, today while exploring GitHub looking for interesting projects found yours and I would love to contribute by designing a logo for it. I'm a designer in development and an open source enthusiast. We would be working together to create the design that fits best. Of course it's totally free and you can quit whenever you want. In case you agree, please let me know any preferences you may have about colors, shapes, etc.

Kind regards.

**Comment by grzuy** [comment]
Hi @jorlogicus,

Thank you very much for offering help with that!

Making that decision is out of my scope, so I'll leave that to someone from Kickstarter (cc/ @frewsxcv).
